opengrok-indexer/src/main/java/org/opengrok/indexer/web/Util.java

```diff
@@ -743,16 +744,8 @@ public static void readableLine(int num, Writer out, Annotation annotation,
             if (enabled) {
                 out.write(anchorClassStart);
                 out.write("r");
-                if (annotation.getFileVersion(r) != 0) {
-                    /*
-                        version number, 1 is the most recent
-                        generates css classes version_color_n
-                     */
-                    int versionNumber = Math.max(1,
-                            annotation.getFileVersionsCount()
-                            - annotation.getFileVersion(r) + 1);
-                    out.write(" version_color_" + versionNumber);
-                }
+                out.write("\" style=\"background-color: ");
+                out.write(annotation.getColors().getOrDefault(r, "inherit"));
```

Comment:
can you avoid calling `getColors()` for every line ?

Reply:
lol, yes

Reply:
used lazy instantiation